lib/goodcity/health_checks/package_stockit_id_nil_check.rb

```diff
@@ -0,0 +1,19 @@
+require 'goodcity/health_checks/base'
+
+module Goodcity
+  class HealthChecks
+
+    class PackageStockitIdNilCheck < Base
+      desc "Packages should contain a stockit_id reference."
+      def run
+        ids = Package.where(stockit_id: nil).pluck(:id)
+        if ids.count == 0
+          pass!
+        else
+          fail_with_message!("GoodCity Packages with nil stockit_id: #{ids.join(', ')}")
+        end
+      end
```

Comment:
Yes I know what you mean. It's a tough call... too much abstraction and it becomes hard to read. Too little and nothing is DRY!

There is a pattern here with some checks where if there are any results found that then is bad and a query resulting in an empty array is good. Perhaps something like:

    query = Package.where(stockit_id: nil)
    fail_if_not_empty(query, failure_message)


    def fail_if_not_empty(query, failure_message = '')
      query.any? ? fail_with_message!(message) : pass!
    end
Could take it a bit further and say 'fail if less than X items' but we don't have that use case yet.